bin/crew

```diff
@@ -1924,7 +1924,7 @@ def reinstall_command(args)
     @pkgName = name
     search @pkgName
     print_current_package
-    @pkg.build_from_source = true if @opt_source || @opt_recursive || (CREW_BUILD_FROM_SOURCE == '1')
+    @pkg.build_from_source = true if @opt_source || @opt_recursive || CREW_BUILD_FROM_SOURCE
```

Comment:
Does `export CREW_BUILD_FROM_SOURCE=false` still build from source?

Reply:
I think we could omit the `true if` too, correct?

Reply:
IIRC this parts means overriding the original value with `true` if `@opt_source || @opt_recursive || CREW_BUILD_FROM_SOURCE` (the `true if` is necessary)

Reply:
> Does `export CREW_BUILD_FROM_SOURCE=false` still build from source?

No, only the exact `1` value will trigger the flag

Reply:
This contradicts what you said earlier about the reason it is necessary to set the environment variable to 1.

Reply:
`@opt_source || @opt_recursive || CREW_BUILD_FROM_SOURCE` will return true anyways, right?

Reply:
> This contradicts what you said earlier about the reason it is necessary to set the environment variable to 1.

Umm, you didn't get my meaning... For example:
```
export CREW_BUILD_FROM_SOURCE=false  # will be `false` in const.rb
export CREW_BUILD_FROM_SOURCE=true   # will be `false` in const.rb
export CREW_BUILD_FROM_SOURCE=""     # will be `false` in const.rb
export CREW_BUILD_FROM_SOURCE=1      # only this will make the const `true`
```

> @opt_source || @opt_recursive || CREW_BUILD_FROM_SOURCE will return true anyways, right?

`CREW_BUILD_FROM_SOURCE` isn't always true...

Reply:
Well, if it isn't true and it doesn't build from source, that is the expected behavior, right?

Reply:
Yes, unless the `crew install -s` option (`@opt_source`) is given